### ISSUE TITLE
Evaluate removal of async-http-client versions

### ISSUE THREAD

**Issue by katmatt** [commercetools/commercetools-jvm-sdk#1411]
Based on the actually used `async-http-client` versions - which should be sent as part of the `user-agent` header - we could consider to remove support for `async-http-client` from our SDK.

**Comment by acbeni** [comment]
@katmatt do you mean removing `sdk-http-ahc-1_8`, `sdk-http-ahc-1_8` ... ? 
they are no longer used, and i think we can safely remove them.
WDYT?

**Comment by katmatt** [comment]
@acbeni Yes, I was mainly thinking about the `sdk-http-ahc-1_` clients. But we should take a look at our request logs and analyze the  http clients are still used by our users. Based on the results we can then decide which clients could be removed.
